airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -766,7 +766,7 @@ private void deleteDestination(UUID destinationId) throws ApiException {
   private static void waitForSuccessfulJob(JobsApi jobsApi, JobRead originalJob) throws InterruptedException, ApiException {
     JobRead job = originalJob;
     int count = 0;
-    while (count < 15 && (job.getStatus() == JobStatus.PENDING || job.getStatus() == JobStatus.RUNNING)) {
+    while (count < 60 && (job.getStatus() == JobStatus.PENDING || job.getStatus() == JobStatus.RUNNING)) {
```

Comment:
I added 45 secs to the timeout just for trying things out... not sure if we could bump in a smaller increment or is this fine?